src/ravl/ravl.c

```diff
@@ -0,0 +1,542 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
please update license and copyright date

Reply:
Done